homeassistant/components/fireservicerota/binary_sensor.py

```diff
@@ -0,0 +1,103 @@
+"""Binary Sensor platform for FireServiceRota integration."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN as FIRESERVICEROTA_DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistantType, entry: ConfigEntry, async_add_entities
+) -> None:
+    """Set up FireServiceRota binary sensor based on a config entry."""
+    coordinator = hass.data[FIRESERVICEROTA_DOMAIN][entry.entry_id]
+
+    async_add_entities([ResponseBinarySensor(coordinator, entry)], True)
```

Comment:
No need to pass `True`. That indicates that the entity needs to update but the data has already been fetched by the update coordinator.

Reply:
Strangely enough without it the binary_sensor only gets the correct state after the minute update_interval after a (re)start.
Have to look into this...